packages/react-menu/src/components/MenuGroupHeader/useMenuGroupHeaderStyles.ts

```diff
@@ -1,19 +1,21 @@
+import * as React from 'react';
 import { mergeClasses, makeStyles } from '@fluentui/react-make-styles';
+import { tokens } from '@fluentui/react-theme';
 import type { MenuGroupHeaderState } from './MenuGroupHeader.types';
 
 export const menuGroupHeaderClassName = 'fui-MenuGroupHeader';
 
 const useStyles = makeStyles({
-  root: theme => ({
-    fontSize: theme.fontSizeBase200,
-    color: theme.colorNeutralForeground3,
+  root: {
+    fontSize: tokens.fontSizeBase200,
+    color: tokens.colorNeutralForeground3,
     paddingLeft: '12px',
     paddingRight: '12px',
-    fontWeight: theme.fontWeightSemibold,
+    fontWeight: tokens.fontWeightSemibold as React.CSSProperties['fontWeight'],
```

Comment:
😢

Reply:
@behowell mentioned this in a separate PR, I'm going to be spinning up a separate issue to address this and be able to remove this casting.

Reply:
@miroslavstastny I've noted this in #21212 and will work it out in a separate PR that addresses that issue.

Reply:
Here's the PR to fix the issue: #21217